pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ComponentBlockDef.java

```diff
@@ -26,6 +26,8 @@
     @SerializedName("hideNumber")
     private boolean hideNumber;
 
+    private transient long revisionId;
```

Comment:
I got a bit confused on why we need this, and then I saw that this is being used in `FormBlockCreatorHelper`. Two things:
* Maybe we should rename this to `componentRevisionId`. I got confused on thinking that this is the "revision" of the block itself...
* If we inspect the code, we see that component revision id is not really used anywhere. Maybe there's a way to not require on this.